src/Codeception/Module/DataFactory.php

```diff
@@ -173,6 +170,16 @@ public function _beforeSuite($settings = [])
             }
         }
     }
+    
+    /**
+     * @return StoreInterface
+     */
+    protected function getStore()
+    {
+        return $this->ormModule instanceof DataMappe
```

Comment:
Incorrect class name for `DataMapper` (`r` is missing, and maybe some test cases since the code passed the current tests).